supplementary/alica_ros1/alica_ros_proxy/include/clock/AlicaRosTimer.h

```diff
@@ -71,6 +70,11 @@ class SyncStopTimerRosImpl : public std::enable_shared_from_this<SyncStopTimerRo
             _userCbInProgress = true;
         }
 
+        if (!_timer.isValid()) {
```

Comment:
why would the timer be invalid here?

Reply:
We can alternatively just add the callback to the global callback queue ourselves to avoid setting duration to 0 and having to rely on strange behaviour